pkg/apis/wordpress/v1alpha1/wordpress_types.go

```diff
@@ -54,8 +54,7 @@ type WordpressSpec struct {
 	Replicas *int32 `json:"replicas,omitempty"`
 	// Domains for which this this site answers.
 	// The first item is set as the "main domain" (eg. WP_HOME and WP_SITEURL constants).
-	// +kubebuilder:validation:MinItems=1
-	Domains []Domain `json:"domains"`
+	Domains []Domain `json:"domains,omitempty"`
```

Comment:
Why do we let creation without at least a domain?

Reply:
This is what we agreed. The default domain is `<wp name>.<namespace>.svc.cluster.local` if not specified.